docs/grpc_disruption.md

```diff
@@ -0,0 +1,96 @@
+# GRPC disruption
+
+The `grpc` field offers a way to inject spoofed gRPC responses on the server-side. To get this disruption to work, you must apply some code changes to the instantiation of your gRPC server.
+
+* `port` is the port exposed on target pods (the target pods are specified in `spec.selector`)
```

Comment:
I'm not sure this flows clearly from the previous line. We were just introducing the code changes to the grpc server needed, when we start explaining all of the spec options